src/app/pages/common/entity/entity-job/entity-job.component.html

```diff
@@ -10,13 +10,21 @@ <h1 mat-dialog-title>{{title | translate}}</h1>
 		<label *ngIf="!hideProgressValue">
 			{{ progressTotalPercent | number: '1.2-2'}}%
 		</label>
-    </div>
+	</div>
+	<div *ngIf="description === 'Uploading file...'">
```

Comment:
perhaps make this more generic so it can be used for other things

Reply:
Latest commit should allow for any message, with default message set to 'Starting job...'